# Japanese translation for apport
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-12-07 16:21+0100\n"
"PO-Revision-Date: 2021-10-19 10:21+0000\n"
"Last-Translator: id:sicklylife <Unknown>\n"
"Language-Team: Japanese <ja@li.org>\n"
"Language: ja\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-08-19 12:20+0000\n"
"X-Generator: Launchpad (build 87919ae1fbf1142b0e02b80650e6ccf3e775beb3)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "システム情報を収集"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr "この問題を報告するには、システム情報を収集するための認証が必要です"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "システムの問題の報告"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr ""
"システムプログラムの問題レポートにアクセスするにはパスワードを入力してくださ"
"い"

#: ../apport/ui.py:165
msgid "This package does not seem to be installed correctly"
msgstr "このパッケージは恐らく正常にインストールされていません"

#: ../apport/ui.py:175
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""
"これは恐らく %s の公式パッケージではありません。利用可能なパッケージのイン"
"デックスを更新してから再度試してみてください。問題が解決しない場合、関連する"
"サードパーティーのパッケージの削除も試してみてください。"

#: ../apport/ui.py:205
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"古いバージョンのパッケージが利用されています。以下のパッケージをアップグレー"
"ドした上で、それでも問題が発生するかを確認してください:\n"
"\n"
"%s"

#: ../apport/ui.py:359
msgid "unknown program"
msgstr "不明なプログラム"

#: ../apport/ui.py:362
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "残念ながら、プログラム \"%s\" が予期せず終了しました"

#: ../apport/ui.py:365 ../apport/ui.py:1899
#, python-format
msgid "Problem in %s"
msgstr "%s に問題があります"

#: ../apport/ui.py:370
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"コンピューターに、問題を自動的に解析して開発者にレポートを送るための十分な空"
"きメモリーがありません。"

#: ../apport/ui.py:430 ../apport/ui.py:448 ../apport/ui.py:621
#: ../apport/ui.py:630 ../apport/ui.py:867 ../apport/ui.py:1677
#: ../apport/ui.py:1860 ../apport/ui.py:1865
msgid "Invalid problem report"
msgstr "無効な問題のレポートです"

#: ../apport/ui.py:431
msgid "You are not allowed to access this problem report."
msgstr "この問題を報告するための権限がありません。"

#: ../apport/ui.py:439
msgid "Error"
msgstr "エラー"

#: ../apport/ui.py:441
msgid "There is not enough disk space available to process this report."
msgstr "このレポートを作成するための空きスペースがディスクにありません。"

#: ../apport/ui.py:479
msgid "No PID specified"
msgstr "PIDが指定されていません"

#: ../apport/ui.py:481
msgid "You need to specify a PID. See --help for more information."
msgstr ""
"PIDを指定する必要があります。詳細については --help の出力を確認してください。"

#: ../apport/ui.py:492 ../apport/ui.py:597
msgid "Invalid PID"
msgstr "無効なプロセスID"

#: ../apport/ui.py:493
msgid "The specified process ID does not exist."
msgstr "指定されたプロセスIDは存在しません。"

#: ../apport/ui.py:498
msgid "Not your PID"
msgstr "プロセスIDの所有者ではありません"

#: ../apport/ui.py:499
msgid "The specified process ID does not belong to you."
msgstr "あなたは指定されたプロセスIDの所有者ではありません。"

#: ../apport/ui.py:554
msgid "No package specified"
msgstr "パッケージが指定されていません"

#: ../apport/ui.py:556
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"パッケージまたはプロセスIDを指定する必要があります。詳細については --help の"
"出力を確認してください。"

#: ../apport/ui.py:584
msgid "Permission denied"
msgstr "許可がありません"

#: ../apport/ui.py:586
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"このプロセスを操作する権限がありません。プロセスの所有者もしくはrootユーザー"
"として実行してください。"

#: ../apport/ui.py:599
msgid "The specified process ID does not belong to a program."
msgstr "指定されたプロセスIDはプログラムに属していません。"

#: ../apport/ui.py:623
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Symptomスクリプト %s は、影響を受けたパッケージを特定していません"

#: ../apport/ui.py:631
#, python-format
msgid "Package %s does not exist"
msgstr "パッケージ %s は存在しません"

#: ../apport/ui.py:660 ../apport/ui.py:872 ../apport/ui.py:905
#: ../apport/ui.py:915
msgid "Cannot create report"
msgstr "レポートを作成できません"

#: ../apport/ui.py:675 ../apport/ui.py:734 ../apport/ui.py:752
msgid "Updating problem report"
msgstr "問題点のレポートを更新中"

#: ../apport/ui.py:677
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"あなたはこの問題の報告者または登録者でないか、報告が重複しているかすでに終了"
"しています。\n"
"\n"
"\"apport-bug\"を使って新規にレポートを作成してください。"

#: ../apport/ui.py:689
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"あなたはこの問題の報告者または登録者ではありません。新しいバグとしてコメント"
"と添付ファイルを提出するより、他のバグと重複したバグとしてマークするほうがよ"
"り簡単です。\n"
"\n"
"その後、\"apport-bug\"を利用して新たにバグ報告を行い、このバグの中で提出する"
"ものについてのコメントを行うことをお勧めします。\n"
"\n"
"本当に進めますか?"

#: ../apport/ui.py:735 ../apport/ui.py:753
msgid "No additional information collected."
msgstr "収集する追加情報はありません。"

#: ../apport/ui.py:816
msgid "What kind of problem do you want to report?"
msgstr "どのような種類の問題を報告しますか？"

#: ../apport/ui.py:837
msgid "Unknown symptom"
msgstr "未知の現象"

#: ../apport/ui.py:838
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "現象 \"%s\" は知られていません。"

#: ../apport/ui.py:874
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:890
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"このメッセージを閉じた後でアプリケーションウィンドウ上でクリックして、その問"
"題について報告してください。"

#: ../apport/ui.py:907 ../apport/ui.py:916
msgid "xprop failed to determine process ID of the window"
msgstr "xpropはウィンドウのプロセスIDを特定できませんでした"

#: ../apport/ui.py:931
#, fuzzy, python-format
msgid "%(prog)s <report number>"
msgstr "%prog <report number>"

#: ../apport/ui.py:932
msgid "Specify package name."
msgstr "パッケージ名を指定"

#: ../apport/ui.py:938 ../apport/ui.py:1070
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "レポートに追加タグを追加します。複数回指定できます。"

#: ../apport/ui.py:976
#, fuzzy, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr "%prog [options] [symptom|pid|package|program path|.apport/.crash file]"

#: ../apport/ui.py:987
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"バグファイリングモードを開始します。実行には --package と追加の --pid を用い"
"るか、あるいはただ --pid だけとするオプションが必要です。どちらも与えられてい"
"ないときには、既知の現象のリストを表示します（暗黙の一つの引数が与えられた場"
"合）。"

#: ../apport/ui.py:996
msgid "Click a window as a target for filing a problem report."
msgstr ""
"問題レポートを埋めるには、ターゲットとなるウィンドウをクリックしてください。"

#: ../apport/ui.py:1005
msgid "Start in bug updating mode. Can take an optional --package."
msgstr "バグ更新モードを開始します。オプション --package が使用できます。"

#: ../apport/ui.py:1014
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"SYMPTOM（症状）についてのバグを報告します（唯一の引数は症状の名前を意味しま"
"す）。"

#: ../apport/ui.py:1023
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"--file-bug モードで使用するパッケージ名を指定します。--pid が指定されている場"
"合はオプション扱いになります（唯一の引数はパッケージ名を意味します）。"

#: ../apport/ui.py:1034
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"--file-bug モードの対象になる実行中のプログラムを指定します。このオプションを"
"指定した場合、バグ報告にはより多くの情報が含まれます（唯一の引数はプロセスID"
"を意味します）。"

#: ../apport/ui.py:1042
msgid "The provided pid is a hanging application."
msgstr "指定されたpidはハングアップしたアプリケーションのものです。"

#: ../apport/ui.py:1050
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"クラッシュの報告を %s で保留中のプログラムのかわりに .apport または .crash "
"ファイルで行う（ファイル名が引数のみで与えられた場合）。"

#: ../apport/ui.py:1060
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"バグファイリングモードでは、収集した情報は報告する代わりにファイルに保存され"
"ます。このファイルは、後で別のコンピューターから報告することができます。"

#: ../apport/ui.py:1078
msgid "Print the Apport version number."
msgstr "Apportのバージョン番号を表示します。"

#: ../apport/ui.py:1240
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr "クラッシュの解析を行うため端末ウィンドウでapport-retraceを起動します。"

#: ../apport/ui.py:1244
msgid "Run gdb session"
msgstr "gdbセッションを実行"

#: ../apport/ui.py:1245
msgid "Run gdb session without downloading debug symbols"
msgstr "デバッグシンボルをダウンロードせずgdbセッションを実行"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1247
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "%s を完全なシンボリックスタックトレースに更新する"

#: ../apport/ui.py:1309
msgid "Can't remember send report status settings"
msgstr "レポートの送信のステータス設定を記憶できません"

#: ../apport/ui.py:1313
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""
"クラッシュレポートの状態の保存に失敗しました。報告モードを「自動」または「し"
"ない」に設定できません。"

#: ../apport/ui.py:1396 ../apport/ui.py:1409
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""
"この問題レポートは、もうインストールされていないプログラムに対するものです。"

#: ../apport/ui.py:1433
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"この問題はプログラム %s に発生したものですが、このプログラムはクラッシュして"
"から変更されています。"

#: ../apport/ui.py:1494 ../apport/ui.py:1618 ../apport/ui.py:1869
msgid "This problem report is damaged and cannot be processed."
msgstr "この問題の報告は破損しており、処理することができません。"

#: ../apport/ui.py:1503
msgid "This report is about a package that is not installed."
msgstr "このレポートは、インストールされていないパッケージに対するものです。"

#: ../apport/ui.py:1511
msgid "An error occurred while attempting to process this problem report:"
msgstr "この問題の報告の処理中にエラーが発生しました:"

#: ../apport/ui.py:1528
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""
"このアプリケーションは2種類のバージョンがインストールされています。どちらのバ"
"グを報告しますか？"

#: ../apport/ui.py:1533
#, python-format
msgid "%s snap"
msgstr "snapの %s"

#: ../apport/ui.py:1534
#, python-format
msgid "%s deb package"
msgstr "debパッケージの %s"

#: ../apport/ui.py:1575
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""
"%s は %s が公開しているsnapです。バグ報告や質問については %s に連絡してくださ"
"い。"

#: ../apport/ui.py:1580
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""
"%s は %s が公開しているsnapです。連絡先のアドレスは提供されていないため、バグ"
"報告や質問については https://forum.snapcraft.io/ にアクセスしてください。"

#: ../apport/ui.py:1679
msgid "Could not determine the package or source package name."
msgstr "パッケージまたはソースパッケージ名を特定できませんでした。"

#: ../apport/ui.py:1704
msgid "Unable to start web browser"
msgstr "ウェブブラウザーを起動できません"

#: ../apport/ui.py:1705
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "%s を開くためのウェブブラウザーを起動できません"

#: ../apport/ui.py:1815
msgid "Network problem"
msgstr "ネットワークの問題"

#: ../apport/ui.py:1819
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"クラッシュデータベースに接続できませんでした。インターネット接続を確認してく"
"ださい。"

#: ../apport/ui.py:1851
msgid "Memory exhaustion"
msgstr "メモリー不足"

#: ../apport/ui.py:1853
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"コンピューターには、このクラッシュレポートを処理するのに十分なメモリーがあり"
"ません。"

#: ../apport/ui.py:1904
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"問題を報告することができません:\n"
"\n"
"%s"

#: ../apport/ui.py:1960 ../apport/ui.py:1972
msgid "Problem already known"
msgstr "問題は既知のものです"

#: ../apport/ui.py:1962
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"この問題は、ブラウザーに表示されたバグレポートですでに報告されています。開発"
"者にとって役立つような、より詳しい情報を追加できるかどうか確認してください。"

#: ../apport/ui.py:1974
msgid "This problem was already reported to developers. Thank you!"
msgstr "この問題はすでに開発者に報告されています。ありがとうございます！"

#: ../bin/apport-cli.py:84
msgid "Press any key to continue..."
msgstr "続けるには何かキーを押してください..."

#: ../bin/apport-cli.py:91
msgid "What would you like to do? Your options are:"
msgstr "対処方法を選択してください:"

#: ../bin/apport-cli.py:104
#, python-format
msgid "Please choose (%s):"
msgstr "選択してください (%s):"

#: ../bin/apport-cli.py:172
#, python-format
msgid "(%i bytes)"
msgstr "(%i バイト)"

#: ../bin/apport-cli.py:174 ../gtk/apport-gtk.py:169 ../kde/apport-kde.py:406
msgid "(binary data)"
msgstr "(バイナリデータ)"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.py:208 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:199
msgid "Send problem report to the developers?"
msgstr "問題のレポートを開発者に送信しますか？"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"問題のレポートが送信された後で、フォームに入力してください（入力用ページを"
"ウェブブラウザーで自動的に開きます）。"

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "レポートの送信(&S) (%s)"

#: ../bin/apport-cli.py:223
msgid "&Examine locally"
msgstr "ローカルで解析する(&E)"

#: ../bin/apport-cli.py:227
msgid "&View report"
msgstr "レポートの表示(&V)"

#: ../bin/apport-cli.py:230
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"後で送信もしくは他の場所にコピーするためにレポートファイルを保持する(&K)"

#: ../bin/apport-cli.py:235
msgid "Cancel and &ignore future crashes of this program version"
msgstr "中止し、このプログラムバージョンでは今後クラッシュしても無視する(&I)"

#: ../bin/apport-cli.py:238 ../bin/apport-cli.py:332 ../bin/apport-cli.py:366
#: ../bin/apport-cli.py:389
msgid "&Cancel"
msgstr "キャンセル(&C)"

#: ../bin/apport-cli.py:273
msgid "Problem report file:"
msgstr "問題報告ファイル:"

#: ../bin/apport-cli.py:279 ../bin/apport-cli.py:284
msgid "&Confirm"
msgstr "確認(&C)"

#: ../bin/apport-cli.py:283
#, python-format
msgid "Error: %s"
msgstr "エラー: %s"

#: ../bin/apport-cli.py:289 ../kde/apport-kde.py:462
msgid "Collecting problem information"
msgstr "問題の情報を集めています"

#: ../bin/apport-cli.py:291
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"収集された情報はアプリケーションを改善するために開発者へ送られます。\n"
"しばらくお待ちください。"

#: ../bin/apport-cli.py:307 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:511
msgid "Uploading problem information"
msgstr "問題の情報をアップロードしています"

#: ../bin/apport-cli.py:309
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"収集された情報はバグトラッキングシステムに送信されます。\n"
"しばらくお待ちください。"

#: ../bin/apport-cli.py:365
msgid "&Done"
msgstr "完了(&D)"

#: ../bin/apport-cli.py:371
msgid "none"
msgstr "なし"

#: ../bin/apport-cli.py:373
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "現在の設定: %s。複数選択:"

#: ../bin/apport-cli.py:390
msgid "Choices:"
msgstr "選択肢:"

#: ../bin/apport-cli.py:404
msgid "Path to file (Enter to cancel):"
msgstr "ファイルへのパス (Enterでキャンセル):"

#: ../bin/apport-cli.py:410
msgid "File does not exist."
msgstr "ファイルが存在しません。"

#: ../bin/apport-cli.py:412
msgid "This is a directory."
msgstr "これはディレクトリです。"

#: ../bin/apport-cli.py:418
msgid "To continue, you must visit the following URL:"
msgstr "続行するには、以下のURLを開いてください:"

#: ../bin/apport-cli.py:421
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"ブラウザーを今すぐ起動するか、または他のコンピューターのブラウザーにURLをコ"
"ピーしてください。"

#: ../bin/apport-cli.py:427
msgid "Launch a browser now"
msgstr "今すぐブラウザーを起動する"

#: ../bin/apport-cli.py:443
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"未解決のクラッシュレポートはありません。詳しくは --help を試してみてくださ"
"い。"

#: ../bin/apport-retrace.py:44
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr ""
"これ以上レポートに新しいトレースを書き込みませんが、標準出力には表示します。"

#: ../bin/apport-retrace.py:53
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"レポートのコアダンプに対してインタラクティブなgdbセッションを開始します（-o "
"無視；レポートを上書きしません）"

#: ../bin/apport-retrace.py:62
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"オリジナルのレポートを変更せずに、指定されたファイルに変更後のレポートを書き"
"込みます"

#: ../bin/apport-retrace.py:72
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr "スタックトレースを再生成した後にコアダンプをレポートから取り除きます"

#: ../bin/apport-retrace.py:80
msgid "Override report's CoreFile"
msgstr "レポートのコアファイルを上書きします"

#: ../bin/apport-retrace.py:86
msgid "Override report's ExecutablePath"
msgstr "レポートの ExecutablePath を上書きします"

#: ../bin/apport-retrace.py:92
msgid "Override report's ProcMaps"
msgstr "レポートの ProcMaps を上書きします"

#: ../bin/apport-retrace.py:98
msgid "Rebuild report's Package information"
msgstr "レポートに含まれるパッケージ情報を再生成します"

#: ../bin/apport-retrace.py:105
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"一時的なサンドボックスを作成し、必要なパッケージとデバッグシンボルをそこにダ"
"ウンロードおよびインストールします。このオプションが指定されなかった場合に"
"は、必要なパッケージやデバッグシンボルはすでにお使いのシステムにインストール"
"されているものとして処理を続行します。引数はパッケージングシステムの設定の"
"ベースディレクトリを指定します。「system」を指定した場合にはシステムの設定"
"ファイルを利用しますが、その場合には現在実行中のリリースで発生したクラッシュ"
"しかリトレースできません。"

#: ../bin/apport-retrace.py:119
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""
"gdbとその依存関係をインストールする一時的なサンドボックスをビルドし、インス"
"トールされているgdbのバージョンとは関係なく、レポートにあるのと同じリリースを"
"利用します"

#: ../bin/apport-retrace.py:129
msgid "Report download/install progress when installing packages into sandbox"
msgstr ""
"サンドボックスにパッケージをインストールする際に、ダウンロードおよびインス"
"トールの進捗状況を表示します"

#: ../bin/apport-retrace.py:136
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "ログメッセージの先頭にタイムスタンプを挿入します（バッチ処理用）"

#: ../bin/apport-retrace.py:142
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""
"レポートで指定された提供元から、サードパーティーのリポジトリを作成して使用し"
"ます。"

#: ../bin/apport-retrace.py:150
msgid "Cache directory for packages downloaded in the sandbox"
msgstr ""
"サンドボックスにダウンロードされたパッケージのためのキャッシュディレクトリ"

#: ../bin/apport-retrace.py:156
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"展開したパッケージのためのディレクトリ。これ以降はダウンロード済みのパッケー"
"ジはすべてこのサンドボックスに展開されているものと仮定します。"

#: ../bin/apport-retrace.py:167 ../bin/apport-valgrind.py:97
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr "サンドボックスに追加のパッケージをインストールします（複数回指定可）"

#: ../bin/apport-retrace.py:174
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"クラッシュデータベース認証情報のファイルへのパス。これはリトレースしたスタッ"
"クトレースをアップロードする際にクラッシュIDを指定するために使われます（-g, -"
"o, または -s が指定されていない場合）。"

#: ../bin/apport-retrace.py:184
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"スタックトレースのリトレースを表示し、クラッシュデータベースに送る前に確認し"
"ます。"

#: ../bin/apport-retrace.py:192
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr "sqliteデータベースの複製へのパス（デフォルトは複製のチェックをしない）"

#: ../bin/apport-retrace.py:200
msgid "Do not add StacktraceSource to the report."
msgstr "レポートにスタックトレースソースを追加しないでください。"

#: ../bin/apport-retrace.py:213
msgid "You cannot use -C without -S. Stopping."
msgstr "-S を指定せずに -C を使用することはできません。中止します。"

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:243
msgid "OK to send these as attachments? [y/n]"
msgstr "これらを添付ファイルとして送ってよろしいですか？ [y/n]"

#: ../bin/apport-unpack.py:29
#, fuzzy, python-format
msgid "%(prog)s <report> <target directory>"
msgstr "使用法 : %s <レポート> <対象ディレクトリ>"

#: ../bin/apport-unpack.py:31
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:33
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:59
msgid "Destination directory exists and is not empty."
msgstr "目的のディレクトリは存在しますが、空ではありません。"

#: ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "詳細はmanページをご覧ください。"

#: ../bin/apport-valgrind.py:46
msgid "specify the log file name produced by valgrind"
msgstr "valgrindによって生成されたログファイルの名前を指定"

#: ../bin/apport-valgrind.py:52
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"以前作成されたサンドボックスディレクトリ (SDIR) を再利用するか、もし存在しな"
"い場合には作成する"

#: ../bin/apport-valgrind.py:60
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"追加のデバッグシンボル用にサンドボックスディレクトリを作成または再利用せず、"
"インストールされたデバッグシンボルのみを利用する。"

#: ../bin/apport-valgrind.py:69
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"以前作成されたキャッシュディレクトリ (CDIR) を再利用するか、もし存在しない場"
"合には作成する"

#: ../bin/apport-valgrind.py:78
msgid "report download/install progress when installing packages into sandbox"
msgstr ""
"サンドボックス内にパッケージをインストールする際、ダウンロードおよびインス"
"トールの進捗状況を報告する"

#: ../bin/apport-valgrind.py:86
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:129
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "エラー: %s は実行ファイルではありません。中止します。"

#: ../data/apportcheckresume.py:68
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""
"このことは前のサスペンド中に発生しており、システムが正常に復帰できませんでし"
"た。"

#: ../data/apportcheckresume.py:73
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""
"このことは前のハイバネーション中に発生しており、システムが正常に復帰できませ"
"んでした。"

#: ../data/apportcheckresume.py:81
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"リジューム処理が終わる寸前でハングしましたが、通常通り完了していると思われま"
"す。"

#: ../data/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"システムが不安定になる可能性があるので、再起動する必要があるかもしれません。"

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "問題を報告する..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "開発者に異常動作を報告してください"

#: ../gtk/apport-gtk.py:185
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "残念ながら、アプリケーション %s が予期せず停止しました。"

#: ../gtk/apport-gtk.py:188
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "残念ながら、%s が予期せず終了しました。"

#: ../gtk/apport-gtk.py:193 ../kde/apport-kde.py:214 ../kde/apport-kde.py:266
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "残念ながら、%s で内部エラーが発生しました。"

#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:207
msgid "Send"
msgstr "送信"

#: ../gtk/apport-gtk.py:234 ../gtk/apport-gtk.py:666 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:330
msgid "Show Details"
msgstr "詳細を表示"

#: ../gtk/apport-gtk.py:247 ../kde/apport-kde.py:258 ../kde/apport-kde.py:276
msgid "Continue"
msgstr "続行"

#: ../gtk/apport-gtk.py:271
#, python-format
msgid "The application %s has stopped responding."
msgstr "アプリケーション %s は応答していません。"

#: ../gtk/apport-gtk.py:275
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "プログラム \"%s\" は応答していません。"

#: ../gtk/apport-gtk.py:292 ../kde/apport-kde.py:222
#, python-format
msgid "Package: %s"
msgstr "パッケージ: %s"

#: ../gtk/apport-gtk.py:298 ../kde/apport-kde.py:229
msgid "Sorry, a problem occurred while installing software."
msgstr "残念ながら、ソフトウェアのインストール中に問題が発生しました。"

#: ../gtk/apport-gtk.py:310 ../gtk/apport-gtk.py:334 ../kde/apport-kde.py:238
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "アプリケーション %s で内部エラーが発生しました。"

#: ../gtk/apport-gtk.py:316 ../kde/apport-kde.py:245
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "アプリケーション %s が予期せず終了しました。"

#: ../gtk/apport-gtk.py:347 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:271
msgid "If you notice further problems, try restarting the computer."
msgstr "さらに問題が発生する場合は、コンピューターを再起動してみてください。"

#: ../gtk/apport-gtk.py:352 ../kde/apport-kde.py:278
msgid "Ignore future problems of this type"
msgstr "今後、この種類の問題は無視する"

#: ../gtk/apport-gtk.py:670 ../kde/apport-kde.py:327
msgid "Hide Details"
msgstr "詳細を隠す"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:552 ../kde/apport-kde.py:595
msgid "Apport"
msgstr "Apport"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "キャンセル"

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "OK"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "クラッシュレポート"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>残念ながら、内部エラーが発生しました。</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr "この選択を記憶する"

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "今後このバージョンのプログラムの問題を無視する"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr "このアプリケーションを再起動する"

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "ローカルで解析する(_E)"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr "送信しない"

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>問題の情報を集めています</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"あなたの報告する問題を開発者が修正するのに、手助けとなるような情報を収集しま"
"す。"

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>問題の情報をアップロードしています</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:513
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"収集された情報はバグトラッキングシステムに送信されます。しばらくお待ちくださ"
"い。"

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "クラッシュファイルを Apport する"

#: ../kde/apport-kde.py:261
msgid "Leave Closed"
msgstr "閉じて終了する"

#: ../kde/apport-kde.py:262 ../kde/apport-kde.py:432
msgid "Relaunch"
msgstr "再起動する"

#: ../kde/apport-kde.py:355
msgid "Username:"
msgstr "ユーザー名:"

#: ../kde/apport-kde.py:356
msgid "Password:"
msgstr "パスワード:"

#: ../kde/apport-kde.py:461
msgid "Collecting Problem Information"
msgstr "問題の情報を集めています"

#: ../kde/apport-kde.py:464
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"収集された情報はアプリケーションを改善するために開発者へ送られます。しばらく"
"お待ちください。"

#: ../kde/apport-kde.py:510
msgid "Uploading Problem Information"
msgstr "問題の情報をアップロードしています"
